bin/region_mover.rb

```diff
@@ -493,11 +493,29 @@ def getFilename(options, targetServer, port)
   return filename
 end
 
+def getServersRSGroup(rsgroup_admin, hostname, port)
+  rsgroup = rsgroup_admin.getRSGroupOfServer(Address.fromParts(hostname,
+                                             java.lang.Integer.parseInt(port)))
+  return rsgroup unless rsgroup.nil?
+  begin
+    regions = getRegions(getConfiguration(), hostname)
```

Comment:
Does this work? I believe the call to `getRegions` requires a `ServerName` object, and here we just have a string of the `hostname` (with no `port` or `startcode`.) Compare to https://github.com/apache/hbase/blob/1813237699a72f666217c352525c6736781fe3d9/bin/region_mover.rb#L319-L330 where we call `stripServer` to get the host's `ServerName` object from the call to `getServers` which returns a collection of `ServerName`.

Reply:
Fixed this in the latest commit. But I have one more question. It seems that we don't have UTs like java code for rb scripts. How could we test it currently?

Reply:
I too was wondering whether there was any test code.

Reply:
As of today, we don't have any particular UT to validate region_mover functionality on branch-1 (one of the primary reasons to move region_mover logic to RegionMover class in branch-2), hence we need to validate it locally by running region_mover.rb (with RSGroup enabled for this change at least).